### ISSUE TITLE
Publishers CRM 1.1

### ISSUE THREAD

**Issue by yachtcaptain23** [brave-intl/publishers#1317]
Product document: https://docs.google.com/document/d/1DV45oXHEN-ddA_m40nMb3hGMSYOrOWSl_6B7qsYhPdQ/edit?pli=1#heading=h.q9ymf16so9zw

# Definitions

- Invoice - How much the partner is estimating to be paid
- _(Edit February 6: Report and Billing are no longer relevant)_
- <strike>Report - Number of visits, conversions, etc. which leads to the invoiced number above</strike>
- <strike>Billing - How much Brave is paying to the partner</strike>

# Action Items
## Generic Action items
- [x] Create role: Partner. https://github.com/brave-intl/publishers/issues/1327
- [x] UserGroups to allow sign-ins from different emails: https://github.com/brave-intl/publishers/issues/1325
- [x] Allow private file uploads: https://github.com/brave-intl/publishers/issues/1330

### Roles
- [x] Allow assigning of a role to a Partner https://github.com/brave-intl/publishers/issues/1328
- [x] Send email for creating a Partner email https://github.com/brave-intl/publishers/issues/1329
- [x] Specify permissions for a Partner Organization https://github.com/brave-intl/publishers/issues/1351 (Blocked by #1325)
### Upgrade existing Infinite Code Project
- [x] Review reports and invoices: #1333 (Partially blocked by #1330)
- <strike> [ ] Billings tracking: #1334 (Partially blocked by #1330)</strike> - _(Edit February 6: Report and Billing are no longer relevant)_
- <strike> [ ] UI for billing uploads: #1541 </strike> - _(Edit February 6: Report and Billing are no longer relevant)_
### Payouts
- <strike> [ ] Allow for one-time payments https://github.com/brave-intl/publishers/issues/1378</strike>

## Publisher-Partner Page Action Items:
- [ ] ~Separate dashboard page for Publisher-Partner~: #1433 (Not required for CRM 1.1)
- [x] Separate referral page: #1437 
- [x] Separate nav bar: #1559
- [x] Separate payments page: #1531 
- [x] Allow payment via Uphold or not. (Flag is specified by a Brave Business Development Admin)
- [x] Upload reports and invoices (if enabled by a Brave Business Development Admin)
- [x] Updated code creation with limits set by a Brave Business Development Admin

CRM-Later: 
- [ ] <strike> Allow admin to specify negotiated contract agreements https://github.com/brave-intl/publishers/issues/1571 </strike>
- [ ] Lower priority: HTML referral page editing (possible WYSIWIG)
- [ ] Lower priority: Include funnel analytics

**Comment by yachtcaptain23** [comment]
All tasks completed or marked as no longer necessary. Closing.
